packages/client/src/workflow-client.ts

```diff
@@ -487,7 +490,14 @@ export class WorkflowClient extends BaseClient {
       try {
         res = await this.workflowService.getWorkflowExecutionHistory(req);
       } catch (err) {
-        this.rethrowGrpcError(err, { workflowId, runId }, 'Failed to get Workflow execution history');
+        this.rethrowGrpcError(
```

Comment:
Why?

Reply:
Just got autoformatted like that. Not sure how it wasn't before.

Reply:
Yeah but the CI formatter doesn't correct it. :shrug:

Reply:
Can you please reset this file, there's no relevant change here.